components/developer/swig/Makefile

```diff
@@ -79,23 +79,17 @@ COMPONENT_TEST_TARGETS = \
         partialcheck-python-test-suite \
         partialcheck-ruby-test-suite
 
-# common targets
-build:		$(BUILD_32_and_64)
-
-install:	$(INSTALL_32_and_64)
-
-test:		$(TEST_32_and_64)
-
-REQUIRED_PACKAGES += developer/java/openjdk7
+# Build dependencies
+PERL_REQUIRED_PACKAGES += runtime/perl
+REQUIRED_PACKAGES += developer/java/openjdk11
```

Comment:
We only have runtime/java/openjdk11

Reply:
@AndWac Basically I think most components can be fixed using 'gmake update' and some of them can be made 64-bit only.